### DISCUSSION TITLE
xmake use vcpkg

### DISCUSSION THREAD

**Discussion by wangqiongSE** [xmake-io/xmake#3751] [Q&A]
1. 安装vcpkg成功
   `[root@localhost vcpkg]# vcpkg list
  abseil:x64-linux                                  20230125.0#1        an open-source collection designed to augment th...
  c-ares:x64-linux                                  1.19.0              A C library for asynchronous DNS requests
  grpc:x64-linux                                    1.51.1              An RPC library and framework
  grpc[codegen]:x64-linux                                               Build code generator machinery
  openssl:x64-linux                                 3.1.0#4             OpenSSL is an open source project that provides ...
  protobuf:x64-linux                                3.21.12             Protocol Buffers - Google's data interchange format
  re2:x64-linux                                     2023-02-01          RE2 is a fast, safe, thread-friendly alternative...
  upb:x64-linux                                     2022-06-21          μpb (often written 'upb') is a small protobuf i...
  upb[codegen]:x64-linux                                                Build code generator machinery
  vcpkg-cmake-config:x64-linux                      2022-02-06#1
  vcpkg-cmake-get-vars:x64-linux                    2023-03-02
  vcpkg-cmake:x64-linux                             2022-12-22
  zlib:x64-linux                                    1.2.13              A compression library
  `
2. xmake.lua add_requires("vcpkg::grpc",{system=true})
     报错: note: the following packages were not found on your system, try again after installing them:
  -> vcpkg::grpc latest
是需要设置vcpkg packages的地址吗

**Comment by waruqi** [comment]
走 `add_requires("vcpkg::grpc")` 安装，不要自己单独安装，static/runtime/configs 不一定完全匹配。
